src/lib/core/events.ts

```diff
@@ -0,0 +1,23 @@
+import events from "events";
+
+const COMMAND_PREFIX = "command";
+
+class EventEmitter extends events.EventEmitter {
```

Comment:
So **actually** since this is not just an event emitter but really more like a custom event bus, I believe we should have this named `EventBus` or something like that.

This can be done in a follow up PR though.

Reply:
Also, while we're at it... how do you feel about not extending `EventEmitter` and instead have our event emitter delegate calls to nodes `EventEmitter` ?